packages/components/file-uploader-input/src/utils/index.js

```diff
@@ -0,0 +1,21 @@
+import { Constants } from '@tradeshift/elements';
+import componentMessages from './file-uploader-input.yaml';
+export {default as selectors, classNames} from './selectors';
+
+export const Messages = {
```

Comment:
I am not sure it should be lowercase or others should be uppercase

Reply:
Yeah you are right! I'll keep it consistent 👍